resources/views/layouts/dashboard.blade.php

```diff
@@ -22,26 +24,76 @@
     <!-- Sweet alert 2 -->
     <script src="//cdn.jsdelivr.net/npm/sweetalert2@11"></script>
 
+    @php
```

Comment:
big ass copy paste, you can also make a file and load it @include it
you can leave it like this for now, i may change this layout